Source/WebCore/loader/cache/CachedResourceLoader.cpp

```diff
@@ -1507,8 +1505,9 @@ bool CachedResourceLoader::shouldDeferImageLoad(const URL& url) const
 void CachedResourceLoader::reloadImagesIfNotDeferred()
 {
     for (auto& resource : m_documentResources.values()) {
-        if (is<CachedImage>(*resource) && resource->stillNeedsLoad() && clientDefersImage(resource->url()) == ImageLoading::Immediate)
-            downcast<CachedImage>(*resource).load(*this);
+        CachedResourceHandle image =  dynamicDowncast<CachedImage>(*resource);
```

Comment:
Nit: Extra whitespace here between equal sign at `dynamicDowncast`

Reply:
Oops. I merged without fixing this. I'll follow-up.